java/src/jmri/jmrit/beantable/SensorTableAction.java

```diff
@@ -342,16 +342,16 @@ private void canAddRange(ActionEvent e) {
             // Tab All or first time opening, default tooltip
             connectionChoice = "TBD";
         }
-        if (senManager.getClass().getName().contains("ProxySensorManager")) {
-            jmri.managers.ProxySensorManager proxy = (jmri.managers.ProxySensorManager) senManager;
+        if (InstanceManager.sensorManagerInstance().getClass().getName().contains("ProxySensorManager")) {            
+            jmri.managers.ProxySensorManager proxy = (jmri.managers.ProxySensorManager) InstanceManager.sensorManagerInstance();
             List<Manager<Sensor>> managerList = proxy.getDisplayOrderManagerList();
             String systemPrefix = ConnectionNameFromSystemName.getPrefixFromName(connectionChoice);
             for (Manager<Sensor> mgr : managerList) {
                 if (mgr.getSystemPrefix().equals(systemPrefix)) {
                     rangeBox.setEnabled(((SensorManager) mgr).allowMultipleAdditions(systemPrefix));
                     // get tooltip from ProxySensorManager
                     addEntryToolTip = mgr.getEntryToolTip();
-                    log.debug("S add box enabled1");
+                    if (log.isDebugEnabled()) { log.debug("S add box enabled1"); }
```

Comment:
again... Is there a reason you keep adding this?

Reply:
@silverailscolo 

At this point, I consider this a stylistic choice.  It doesn't impact the time it takes for the code to run for a test such as this one.

Back in the old days, not having the if statement would actually impact performance because the arguments for the log.debug() call were always evaluated even if they didn't need to be evaluated.

That said, I do agree with you in this case, but for a non-stylistic reason.  I have been removing if statements protecting a single log statement because they unnecessarily add to the code complexity,  which impacts the test coverage statistics.